docs/resize-rbd.md

```diff
@@ -0,0 +1,277 @@
+/dev/rbd0       9.9G  4.5M  9.8G   1% /var/lib/www/html
+tmpfs           1.9G   12K  1.9G   1% /run/secrets/kubernetes.io/serviceaccount
+tmpfs           1.9G     0  1.9G   0% /proc/acpi
+tmpfs           1.9G     0  1.9G   0% /proc/scsi
+tmpfs           1.9G     0  1.9G   0% /sys/firmware
+#
+```
+
+now you  can see the size of `/var/lib/www/html` is updated from 976M to 9.9G
+
+## Resize Block PVC
+
+```bash
+[$]kubectl get pvc raw-block-pvc -o yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  annotations:
+    pv.kubernetes.io/bind-completed: "yes"
+    pv.kubernetes.io/bound-by-controller: "yes"
+    volume.beta.kubernetes.io/storage-provisioner: rbd.csi.ceph.com
+  creationTimestamp: "2019-12-19T05:56:02Z"
+  finalizers:
+  - kubernetes.io/pvc-protection
+  name: raw-block-pvc
+  namespace: default
+  resourceVersion: "6370"
+  selfLink: /api/v1/namespaces/default/persistentvolumeclaims/raw-block-pvc
+  uid: 54885275-7ca9-4b89-8e7e-c99f375d1174
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: csi-rbd-sc
+  volumeMode: Block
+  volumeName: pvc-54885275-7ca9-4b89-8e7e-c99f375d1174
+status:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 1Gi
+  phase: Bound
+```
+
+Device size in pod using this PVC
+
+```bash
+[$]kubectl exec -it pod-with-raw-block-volume sh
+sh-4.4# blockdev --getsize64 /dev/xvda
+1073741824
+```
+
+rbd Block PVC is mounted at `/dev/xvda` of the pod, and the size is `1073741824`
+bytes which is equal to `1Gib`
+
+Now resize the PVC by editing the PVC (pvc.spec.resource.requests.storage)
+
+```bash
+[$]kubectl edit pvc raw-block-pvc
+
+```
+
+Check PVC status after  editing the pvc storage
+
+```console
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  annotations:
+    pv.kubernetes.io/bind-completed: "yes"
+    pv.kubernetes.io/bound-by-controller: "yes"
+    volume.beta.kubernetes.io/storage-provisioner: rbd.csi.ceph.com
+  creationTimestamp: "2019-12-19T05:56:02Z"
+  finalizers:
+  - kubernetes.io/pvc-protection
+  name: raw-block-pvc
+  namespace: default
+  resourceVersion: "7923"
+  selfLink: /api/v1/namespaces/default/persistentvolumeclaims/raw-block-pvc
+  uid: 54885275-7ca9-4b89-8e7e-c99f375d1174
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: rook-ceph-block
+  volumeMode: Block
+  volumeName: pvc-54885275-7ca9-4b89-8e7e-c99f375d1174
+status:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 1Gi
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: "2019-12-19T06:02:15Z"
+    message: Waiting for user to (re-)start a pod to finish file system resize of
+      volume on node.
+    status: "True"
+    type: FileSystemResizePending
+  phase: Bound
+
+```
+
+Now you can see the pvc stats as FileSystemResizePending, once the kubelet calls
+the NodeExpandVolume to resize the PVC on node, the status conditions will be updated
```

Comment:
change the status conditions will be updated and status will be updated **to** the status conditions will be updated and status.capacity.storage will be updated